website/src/releases/2020-10/migrating-apollo-v3.mdx

```diff
@@ -0,0 +1,103 @@
+---
+date: 2020-10-12
+title: Migrating to the new Apollo Client 3.0
```

Comment:
FYI: since we're planning other breaking changes to be included in v17, it's likely that we adjust this release note to be about migrating to v17 (which obviously includes the apollo client changes among other things).